src/test/java/uk/gov/pay/adminusers/resources/UserResourceAssignServiceAndRoleToUserIT.java

```diff
@@ -31,19 +34,16 @@ void setUp() throws Exception {
     }
     
     @Test
-    void shouldSuccess_whenAddServiceRoleForUser() {
+    void should_assign_service_and_role_to_user_successfully() {
```

Comment:
ah, yet another test naming style - this is the first time I had seen a test_name_with_underscores_between_every_word but I see it's used frequently in adminusers codebase!

Reply:
that might be because of me 😆